pkg/apis/core/validation/validation.go

```diff
@@ -5616,6 +5616,37 @@ func ValidatePodResize(newPod, oldPod *core.Pod, opts PodValidationOptions) fiel
 		allErrs = append(allErrs, field.Forbidden(specPath, "Pod running on node without support for resize"))
 	}
 
+	// Do not allow adding, removing, re-ordering, or renaming containers on resize.
```

Comment:
consider moving this to a helper function

Reply:
done